contracts/types/CommonStruct.sol

```diff
@@ -59,8 +59,8 @@ struct PCKCertTCB {
 }
 
 struct Output {
-    uint16 quoteVersion; // BE
-    bytes4 tee; // BE
+    uint16 quoteVersion; // BE (we should change this)
```

Comment:
I don't think we want to change this. We will keep `quoteVersion` big-endian encoded, because that is how it's represented in the EVM.